### DISCUSSION TITLE
Unsure how to import `from numpy.random import randint` with strict type checking

### DISCUSSION THREAD

**Discussion by NathanCQC** [microsoft/pyright#7248] [Q&A]
The internal structure of this numpy module is complicated and it looks like they have some incomplete type stubs already.

I failed to make my own type stub.

Could you please advise how I should use this module with strict type checking on?

**Comment by erictraut** [comment]
Using `strict` mode can be difficult if you're using libraries that are incomplete from a typing standpoint. 

Make sure you're using the latest version of numpy. The maintainers are fixing typing bugs every release.

If this is still a problem in the latest numpy release, please file bug reports (or even better, PRs) in the numpy project. Problems like this will not be fixed if the maintainers don't know about them.

In the meantime, you may need to disable some of the strict-mode checks or individually suppress diagnostics from strict-mode errors using `# type: ignore` or `# pyright: ignore` comments.
